.env

```diff
@@ -6,3 +6,10 @@ VITE_DEFAULT_NETWORK=1
 VITE_PUSHER_BEAMS_INSTANCE_ID=2e080021-d495-456d-b2cf-84f9fd718442
 VITE_SHUTTER_EON_PUBKEY=0x0e6493bbb4ee8b19aa9b70367685049ff01dc9382c46aed83f8bc07d2a5ba3e6030bd83b942c1fd3dff5b79bef3b40bf6b666e51e7f0be14ed62daaffad47435265f5c9403b1a801921981f7d8659a9bd91fe92fb1cf9afdb16178a532adfaf51a237103874bb03afafe9cab2118dae1be5f08a0a28bf488c1581e9db4bc23ca
 VITE_ENV=develop
+
+# For E2E testing
+CYPRESS_REMOTE_DEBUGGING_PORT=9222
+NETWORK_NAME=mainnet
+SECRET_WORDS='test test test test test test test test test test test junk'
+SKIP_METAMASK_SETUP=false
+DEBUG=synpress:*
```

Comment:
Is it possible to use a different file, maybe a `.env.test` file? would be great if we can keep them seperate 

I found this for jest https://stackoverflow.com/a/70665834/5916893

Reply:
That didn't work but I solved it with `env-cmd -f`